src/components/events/TournamentBracket.tsx

```
@@ -0,0 +1,93 @@
+    { id: 4, round: 1, team1: 'Hawks', team2: 'Falcons', score1: 2, score2: 2, winner: 'Hawks' },
+    // Semifinals
+    { id: 5, round: 2, team1: 'Eagles', team2: 'Bears', score1: 1, score2: 2, winner: 'Bears' },
+    { id: 6, round: 2, team1: 'Sharks', team2: 'Hawks', score1: null, score2: null, winner: null },
+    // Finals
+    { id: 7, round: 3, team1: 'Bears', team2: 'TBD', score1: null, score2: null, winner: null },
+  ]);
+
+  const [isAdmin, setIsAdmin] = useState(false); // Toggle to show organizer tools
+
+  const handleScoreUpdate = (matchId: number, t1: number, t2: number) => {
+    setMatches(matches.map(m => {
```

ESLint

[error] Replace `matches.map(m` with `⏎······matches.map((m)`

(prettier/prettier)